web/templates/dropdown_list.hbs

```diff
@@ -1,15 +0,0 @@
-{{#with item}}
-    <li class="list-item" role="presentation" data-unique-id="{{unique_id}}" data-name="{{name}}" tabindex="0">
-        <a class="dropdown-list-item-common-styles" role="menuitem">
-            {{#if stream}}
-                {{> inline_decorated_stream_name stream=stream show_colored_icon=true}}
-            {{else if is_direct_message}}
-                <i class="zulip-icon zulip-icon-users stream-privacy-type-icon"></i> {{name}}
-            {{else if is_setting_disabled}}
-                <span class="setting-disabled-option"><i class="fa fa-ban" aria-hidden="true"></i>{{t "Disable" }}</span>
-            {{else}}
-                {{name}}
-            {{/if}}
-        </a>
-    </li>
-{{/with}}
```

Comment:
This one is used here:

`web/src/dropdown_widget.js:import render_dropdown_list from "../templates/dropdown_list.hbs";`

Reply:
Oops, I meant to remove `web/templates/settings/dropdown_list.hbs` instead.